pkg/controller/pravegacluster/upgrade_test.go

```diff
@@ -508,6 +865,52 @@ var _ = Describe("Pravega Cluster Version Sync", func() {
 						Ω(rollbackCondition.Message).Should(Equal("0"))
 					})
 				})
+				Context("rollbackClusterVersion error case", func() {
+					var (
+						err          error
+						foundPravega *v1beta1.PravegaCluster
+					)
+					BeforeEach(func() {
+						foundPravega = &v1beta1.PravegaCluster{}
+						r.Reconcile(req)
+						deploy := pravega.MakeControllerDeployment(p)
+						r.client.Create(context.TODO(), deploy)
+						_ = client.Get(context.TODO(), req.NamespacedName, foundPravega)
+						foundPravega.Status.TargetVersion = ""
+						r.client.Update(context.TODO(), foundPravega)
+						err = r.rollbackClusterVersion(foundPravega, "0.6.1")
+					})
+					It("Error should not be nil", func() {
+						Ω(strings.ContainsAny(err.Error(), "failed to get statefulset ()")).Should(Equal(true))
+					})
+				})
+				Context("rollbackConditionFor07", func() {
+					var (
+						result       bool
+						foundPravega *v1beta1.PravegaCluster
+						sts          *appsv1.StatefulSet
+					)
+					BeforeEach(func() {
+						foundPravega = &v1beta1.PravegaCluster{}
+						r.Reconcile(req)
+						sts = &appsv1.StatefulSet{}
+						p.Spec.Version = "0.7.0"
+						sts = pravega.MakeSegmentStoreStatefulSet(p)
+						client.Create(context.TODO(), sts)
+						_, _ = r.Reconcile(req)
+						_ = r.client.Get(context.TODO(), types.NamespacedName{Name: sts.Name, Namespace: p.Namespace}, sts)
+						_ = r.client.Get(context.TODO(), types.NamespacedName{Name: p.Name, Namespace: p.Namespace}, foundPravega)
+						sts.Status.ReadyReplicas = 2
+						*sts.Spec.Replicas = 2
+						r.client.Update(context.TODO(), sts)
+						foundPravega.Spec.Version = "0.5.0"
+						r.client.Update(context.TODO(), foundPravega)
+						result = r.rollbackConditionFor07(foundPravega, sts)
+					})
+					It("Error should not be nil", func() {
```

Comment:
Assertion description is not correct here

Reply:
fixed